diff --git a/dist/emmet-monaco.esm.js b/dist/emmet-monaco.esm.js
index d2af37ba8591f312c1de730f7aada8be7bf9eb6e..96071d042291bb80933bc7f996e18a17e62a2e46 100644
--- a/dist/emmet-monaco.esm.js
+++ b/dist/emmet-monaco.esm.js
@@ -2356,6 +2356,7 @@ function selfClose(config) {
     switch (config.options['output.selfClosingStyle']) {
         case 'xhtml': return ' /';
         case 'xml': return '/';
+        case 'html': return '/';
         default: return '';
     }
 }
@@ -6078,7 +6079,7 @@ function registerProvider(monaco, languages, syntax) {
     }
     const providers = languages.map((language) => monaco.languages.registerCompletionItemProvider(language, {
         triggerCharacters: LANGUAGE_MODES[MAPPED_MODES[language] || language],
-        provideCompletionItems: (model, position) => isValidLocationForEmmetAbbreviation(model, position, syntax, language)
+        provideCompletionItems: (model, position) => syntax === 'html'
             ? doComplete(monaco, model, position, syntax, DEFAULT_CONFIG)
             : undefined,
     }));
